app/controllers/settings/application_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Settings::ApplicationController < ApplicationController
+  layout "settings"
+  before_action :set_settings_menu
+
+  private
+
+  def set_settings_menu
+    # controller_name => array of items
+    @settings_menu = {
+      assistants: Current.user.assistants.ordered.map {
+                    |assistant| [ assistant, edit_settings_assistant_path(assistant) ]
+                  }.to_h,
+      people: { 'Account': edit_settings_person_path }
+    }
```

Comment:
I haven't used this pattern before of a base controller for a set of routes, but it worked out quite nicely.

This particular `@settings_menu` felt a little kludgy to build it up like this, but it seems reasonable enough.